src/plots/polar/layout_attributes.js

```diff
@@ -118,6 +118,7 @@ var radialAxisAttrs = {
     // might need a 'titleside' and even 'titledirection' down the road
 
     hoverformat: axesAttrs.hoverformat,
+    hovertitle: axesAttrs.hovertitle,
```

Comment:
We could set a "hard" `dflt value: `'r'` for `radialaxis.hovertitle` and `'θ'` for `angularaxis.hovertitle`